osquery/tables/networking/windows/interfaces.cpp

```diff
@@ -276,8 +276,11 @@ QueryData genInterfaceAddresses(QueryContext& context) {
 
   const IP_ADAPTER_ADDRESSES* currAdapter = adapters.get();
   while (currAdapter != nullptr) {
-    std::wstring wsAdapterName = std::wstring(currAdapter->FriendlyName);
-    auto adapterName = std::string(wsAdapterName.begin(), wsAdapterName.end());
+    std::string adapterName;
```

Comment:
Can you reserve space here to prevent any reallocations of the push_back operation?

Reply:
Sure, stand by.

Reply:
Done!